src/components/connect.js

```diff
@@ -153,11 +156,23 @@ export default function connect(mapPropsToRequestsToProps, options = {}) {
 
       refetchDatum(prop, mapping) {
         const startedAt = new Date()
+        const isRefresh = mapping.refreshInterval > 0
```

Comment:
not sure i like the name of this variable. i'm not really sure what to call it, but this would still be true on initial page load and could also get confusing with the `refreshing` attribute. Perhaps the clearest would be a more literal `hasRefreshInterval`